app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -76,7 +77,8 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
             val readerModeAction =
                 BrowserToolbar.ToggleButton(
                     image = ContextCompat.getDrawable(requireContext(), R.drawable.ic_readermode)!!,
```

Comment:
Might as well replace all `ContextCompat.getDrawable` in this file so we don't get more bugs

Reply:
@sblatz can you do that on master? Not a big deal for this PR since it's only going to 5.2 branch.